src/lava/proc/dense/models.py

```diff
@@ -102,6 +101,6 @@ def run_spk(self):
         self.a_out.send(self.a_buff)
         s_in = self.s_in.recv()
         a_accum = self.weights[:, s_in].sum(axis=1)
-        self.a_buff = np.left_shift(a_accum,self.weight_exp) if \
-            self.weight_exp > 0 \
-            else np.right_shift(a_accum,-self.weight_exp)
+        self.a_buff = np.left_shift(a_accum,
```

Comment:
Since this stretches over more than one line anyways, a normal if/else statement seems more readable.